src/commands/cluster-meet.json

```diff
@@ -31,7 +31,7 @@
                 "type": "integer"
             },
             {
-                "name": "cluster_bus_port",
```

Comment:
so we're aiming for `-` and not `_` (less convenient if someone wanna name a variable after it).
although they can do the conversion on their own.

Reply:
yes, but we had a strange mix of "-" and "_". i decided to go with "-" because it's more user-friendly when we're talking about rendered syntax

i could change them all to "_" if that's what we want

Reply:
do we have any code that validates that?
let's note that in the top comment.
p.s. i see the previous decision about uniqueness isn't mentioned there correctly.

Reply:
added validation and edited the top comment